libs/common-utils/.env

```diff
@@ -1,96 +0,0 @@
-# PRICE ESTIMATION
```

Comment:
This file was used only for unit-tests. Probably we just should mock .env somehow


<img width="1169" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/a83e4fb4-c952-4aa1-98d1-1359b5ac5c88">

Reply:
I saw only later! I fixed it by using the sam .env for all projects. Now they all share the jest setup